src/openscad.cc

```diff
@@ -887,6 +887,11 @@ int main(int argc, char **argv)
 #endif
   ;
 
+  if constexpr (Feature::HasGuiTesting)
```

Comment:
We've generally used plain #ifdef to enable code enabled by built-time flags. While `if constexpr` is probably a technically better way of solving this, I think keeping things consistent is higher value, just to keep cognitive overhead as low as possible as this file already have a fair bit of complexity.

Changing how we deal with flags in general would be a fun refactoring project though :)

Reply:
Agreed... I answered saying the same just before reading this comment. 
(About reducing the among of #ifdef #endif ... I have made some attempt on MainWindow  with Python)